web/modules/custom/farm_page/src/Controller/FarmPageController.php

```diff
@@ -90,11 +90,12 @@ public function myFarm($uid) {
       }
       else {
         $store = Store::load($store_id);
-        $products = $this->farmService->getFeatureProducts((int) $store_id);
+        $feature_products = $this->farmService->getFeatureProducts((int) $store_id);
         $tags = $this->farmService->getTaxonomyTerm($store->get('field_tags')->getValue());
         $store_images_field = $store->get('field_farm_image')->getValue();
-        $store_images_url = $this->farmService->getImagesUrlFromArray($store_images_field);
-        $full_information_products = $this->farmService->getProductsInfomation($products);
+        $feature_products_info = $this->farmService->getProductsInfomation($feature_products);
+        $categories_id = $store->get('field_products_by_category')->getValue();
+        $products_by_category = $this->farmService->getProductsByCategory($store_id, $categories_id);
```

Comment:
always check isset or empty before load to avoid warning/notice

Reply:
added isset()